specification/metrics/quantityinpricingunit.md

```diff
@@ -0,0 +1,31 @@
+# Quantity in Pricing Unit
+
+The Quantity in Pricing Unit represents the volume of a given resource and/or service used or purchased based on the [Pricing Unit](#pricingunit). This quantity value is used to calculate cost (List, Billed & Effective costs) when multiplied by the associated unit price metric. The Quantity In Pricing Unit pertains to pricing and cost determination and should not be confused with the [Quantity In Usage Unit}(#quantityinusageunit), which is complementary to [Usage Unit](#usageunit) and intended for tracking resource and service consumption.
```

Comment:
1.0 includes the following:

- costs: BilledCost, EffectiveCost, ListCost, NegotiatedCost (optional)
- unit prices: ListUnitPrice, NegotiatedUnitPrice (optional)

Due to this, I recommend 

- **OPTION A:** removing this sentence: `This quantity value is used to calculate cost (List, Billed & Effective costs) when multiplied by the associated unit price metric.`
- **OPTION B:** rephrasing it to something like: `This quantity value can be used to calculate the unit price (e.g., List, Billed, or Effective Unit Price) by dividing the corresponding cost metric.` 

Need just one vote to proceed with the change

Reply:
1

Reply:
A

Reply:
```suggestion
The Quantity in Pricing Unit represents the volume of a given resource and/or service used or purchased based on the [Pricing Unit](#pricingunit). The Quantity In Pricing Unit pertains to pricing and cost, and should not be confused with the [Quantity In Usage Unit](#quantityinusageunit), which is complementary to [Usage Unit](#usageunit) and intended for tracking resource and service consumption.
```